examples/run/cpu/seq_example.cpp

```diff
@@ -96,27 +100,31 @@ int seq_run(const std::string& detector_file, const std::string& cells_dir,
               << std::endl;
     std::cout << "- created " << n_spacepoints << " space points. "
               << std::endl;
-    std::cout << "- created " << n_internal_spacepoints
-              << " internal spacepoints" << std::endl;
     std::cout << "- created " << n_seeds << " seeds" << std::endl;
     return 0;
 }
 
 // The main routine
 //
 int main(int argc, char* argv[]) {
-    if (argc < 4) {
-        std::cout << "Not enough arguments, minimum requirement: " << std::endl;
-        std::cout << "./seq_example <detector_file> <cell_directory> <events>"
-                  << std::endl;
-        return -1;
-    }
-
-    auto detector_file = std::string(argv[1]);
-    auto cell_directory = std::string(argv[2]);
-    auto events = std::atoi(argv[3]);
-
-    std::cout << "Running ./seq_exammple " << detector_file << " "
+    po::options_description desc("Allowed options");
+    desc.add_options()("detector_file", po::value<std::string>()->required(),
+                       "specify detector file");
+    desc.add_options()("cell_directory", po::value<std::string>()->required(),
+                       "specify the directory of cell files");
+    desc.add_options()("events", po::value<int>()->required(),
+                       "number of events");
+
+    po::variables_map vm;
+    po::store(po::parse_command_line(argc, argv, desc), vm);
+    po::notify(vm);
+
+    auto detector_file = vm["detector_file"].as<std::string>();
+    auto cell_directory = vm["cell_directory"].as<std::string>();
+    auto events = vm["events"].as<int>();
+
+    std::cout << "Running ./traccc_seq_example " << detector_file << " "
```

Comment:
You missed this printout. :stuck_out_tongue: (I see that you did change every other one to use `argv[0]`.)

Reply:
I changed that definitely! I just didn't hit ctrl+s...